CONTRIBUTING.md

```diff
@@ -66,6 +67,112 @@ please ensure that:
    deprecation, removal etc) that you have added a reno release note for that
    change and that the PR is tagged for the changelog.
 
+
+### Code review
+
+All code merged to Qiskit, even from maintainers, goes through a code-review
+process after a pull request is made.  There are a small number of
+maintainers who can authorize a final merge, but code review is everyone
```

Comment:
```suggestion
maintainers who can authorize a final merge, but code review involves everyone
```

Reply:
Personally I think "is" is fine here, but I don't mind.  Changed in 724d18de7.